pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -304,6 +305,12 @@ stdenv.mkDerivation {
       done
     ''
 
+    + optionalString (libc != null && targetPlatform.isZ80) ''
+      for isa in z80; do
```

Comment:
Is this supposed to grow?

Reply:
I don't think so. I'll edit this.

Reply:
I'm not even sure it makes sense to modify `bintools-wrapper` like this, at least until the Z80 `libc`issue is resolved.